app/views/state_file/questions/initiate_data_transfer/_data_transfer_buttons.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if @fake_data_transfer_link.present? %>
+  <%= link_to @fake_data_transfer_link.to_s, class: "button button--primary button--icon button--full-width", id: "firstCta" do %>
+    <%= image_tag("icons/link-white.svg", alt: "") %>
+    <%= t(".from_fake_df_page") %>
+  <% end %>
+<% end %>
+
+<% if @irs_df_transfer_link.present? %>
+  <% if Rails.env.production? || Rails.env.staging? %>
```

Comment:
Is there a benefit from doing check here instead of at the concern level?

Reply:
Oh yes I suppose we could find a way to put it in the concern but since its just a front-end change I thought it made more sense to keep it in the view. They are getting the link from the same credential but we just want to add a note about not clicking unless asked for demo but make sure not to show in prod